README.md

```diff
@@ -40,6 +40,29 @@ Then call Ansible:
 
     ANSIBLE_LOAD_CALLBACK_PLUGINS=1 ansible localhost -m ping -vvv
 
+### Puppet
+
+To install and configure a Puppetserver on EL7, run the following:
+
+```bash
+# Install the server - modify as needed for your platform
+yum -y install https://yum.puppet.com/puppet7-release-el-7.noarch.rpm
+yum -y install puppetserver
+# Configure the HTTP report processor
+puppet config set reports store,http
```

Comment:
I had to do this before puppet was available in the environment:

    source /etc/profile.d/puppet-agent.sh

Reply:
Ah yes, somehow I forgot that. As you probably know, it's not needed if you open a fresh shell but it can indeed be something non-obvious. I'll fix it and address the merge conflict at the same time.